package.json

```diff
@@ -19,11 +19,12 @@
   ],
   "scripts": {
     "prebuild": "npm run lint && npm run test:once",
-    "build": "tsc",
+    "build": "tsc && replace-in-file '\"./convex/**/*.*s\"' '\"../../../convex/**/*.*s\"' ./dist/index.js",
```

Comment:
what is this doing / why is it necessary?

Reply:
I have the path hardcoded to work in this repo, and on build I change it to work when installed in node_modules